bots/README.md

```diff
@@ -1,41 +1,69 @@
-# Using Bots
+# Bots Overview
 
 ![Rocket.Chat Bots Banner](./banner.png){:style="width='100%' height='auto'"}
 
-Rocket.Chat supports chatbot and messaging automation integrations, enabling
-chat-ops workflows with multi-channel, multi-user, public and private
-interactions.
+## What are bots in Rocket.Chat?
 
-We are building out support for multiple frameworks, including:
+Bots (or Chatbots) are pieces of software that provide automated messaging in a
+chat platform. They are useful for running pre-defined (scripted) interactions to provide
+support, conversational workflow or on-boarding with multi-channel, multi-user, public and
+private interactions. Bots can also be integrated with external services.
 
-- [bBot](http://bbot.chat/) - complete
-- [Hubot](https://hubot.github.com/) - complete
-- [Botkit](https://botkit.ai/) - in development
-- [Rasa](https://rasa.com/) - in development
-- [Botpress](https://botpress.io/) - development
+In Rocket.Chat, a bot is a special user account with the `bot` role and a specific set
+of permissions.
 
-Some Rocket.Chat server packages implement **internal** bots (e.g.
-[SlackBridge](../administrator-guides/import/slack/slackbridge/)). That type
-of integration is not documented here. See the [Bots FAQ](bots-faq/) for more on
-that distinction.
+## How do bots send and receive messages?
 
-To contribute to features under development see our
-[Bots Project issues](https://github.com/RocketChat/Rocket.Chat/projects/16).
+Rocket.Chat's framework (Meteor) has built-in utilities for streaming updates to data collections.
+Every user in Rocket.Chat has such a collection for their messages. Bots subscribe to
+the collection of messages for their user account, creating a stream that is updated every time a message is sent either directly to them or any room they are joined in.
+
+For more details, see the [Bots Architecture section](bots-architecture/).
+
+## Bots types
+
```

Comment:
Please rework this section.  We do not support internal BOTs at all moving forward.

Perhaps it is good to make a "minor reference" to distinguish between BOTs (always running externally) and Rocket.Chat Apps (always running internally in the server).  Rocket.Chat Apps are in-server extensions or add-on,  designed to add features to the server; and almost NEVER have a conversational interface.

Reply:
Fixed this mentions as discussed